datafusion/proto/proto/datafusion.proto

```diff
@@ -1220,20 +1222,22 @@ message ParquetWriterOptions {
 }
 
 message CsvWriterOptions {
```

Comment:
We have not released a version of DataFusion that contains `CsvWriterOptions` yet, so it is safe to change the field numbers here.